### PR TITLE
Plasmic sync view structure

### PR DESCRIPTION
**Pull request by marcusellison**
### Here I mimic the folder structure you suggest.

What I did:

1. create folder
2. move component into folder
3. change name to index.js
4. change relative import of plasmic files to `../` from `./`
5. update path in plasmic.json to reflect new location of component
6. Run `plasmic sync`
 - We do this to update the import paths of the generated files in 'plasmic' folder, else things break.

### CODE REVIEW

**Comment by xh3b4sd** [comment]
I guess you wanted to chain your PRs so that we see the dedicated changes being made at each stage. What you did now was to branch off of main every time which just shows more and more changes with each of your progression. I cannot tell what happened here looking at the diff.

File: src/component/plasmic/flip_card/PlasmicUpdate.tsx [hunk 1]
```diff
@@ -27,8 +27,8 @@ import {
   StrictProps,
   deriveRenderOpts
 } from "@plasmicapp/react-web";
-import Metric from "../../Metric"; // plasmic-import: _i6uD1XPzdbux6R/component
-import IconButton from "../../IconButton"; // plasmic-import: odPjbfT2kyJgB_S/component
+import Metric from "../../metric/index"; // plasmic-import: _i6uD1XPzdbux6R/component
+import IconButton from "../../iconButton/index"; // plasmic-import: odPjbfT2kyJgB_S/component
```
Comment (xh3b4sd) [inline src/component/plasmic/flip_card/PlasmicUpdate.tsx:31]: Would be cool to get this cleaner as well. 😬 What is actually with the funny comments? Are they essential? 

```suggestion
import metric from "component/metric"; // plasmic-import: _i6uD1XPzdbux6R/component
import iconbutton from "component/iconbutton"; // plasmic-import: odPjbfT2kyJgB_S/component
```

File: src/component/iconButton/index.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import * as React from "react";
 import {
   PlasmicIconButton,
   DefaultIconButtonProps
-} from "./plasmic/flip_card/PlasmicIconButton";
+} from "../plasmic/flip_card/PlasmicIconButton";
```
[the remark this answers is not in the document]
Comment (marcusellison) [reply src/component/iconButton/index.tsx:7]: Yes - we can do this

File: src/component/app/index.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import logo from 'asset/logo.svg'
 
 import './index.css'
-import Update from '../Update'
+import Update from '../update/index'
```
[the remark this answers is not in the document]
Comment (marcusellison) [reply src/component/app/index.tsx:6]: We can't do this. There is a react convention in react where component names have to be capitalized.

[the remark this answers is not in the document]
Comment (xh3b4sd) [reply src/component/app/index.tsx:6]: I meant to have the lower case package name. We talked about this. I updated the suggestion above.

**Pull request merged**